handlers/bin/command-not-found

```diff
@@ -1,77 +1,94 @@
-#!/usr/bin/python
+#!/usr/bin/python3
+
+import gettext
+import os
+import sys
+
+_ = gettext.gettext
 
 EX_NOTFOUND = 127
-SUDO_CMD = "sudo"
-INSTALL_CMD = "zypper install"
+SUDO_CMD = 'sudo'
+INSTALL_CMD = 'zypper install'
+
 
 def print_found(rows):
-    print >> sys.stderr
-    print >> sys.stderr, ngettext("The program '%(prog)s' can be found in the following package:", "The program '%(prog)s' can be found in following packages:", len(rows)) % \
-            ({'prog' : term})
+    print(file=sys.stderr)
```

Comment:
Perhaps just add ``"\n"`` somewhere to the next print statement?

Reply:
no need, as this is exactly the goal of this `print()`

Reply:
Well, I meant to avoid this line completely, but never mind.